GenderPayGap.WebUI/Controllers/Admin/AdminOrganisationScopeController.cs

```diff
@@ -103,7 +103,7 @@ public async Task<IActionResult> ChangeScopePost(long id, int year, AdminChangeS
             };
 
             dataRepository.Insert(newOrganisationScope);
-            await dataRepository.SaveChangesAsync();
+            dataRepository.SaveChangesAsync().Wait();

```

Comment:
Ooh, if you rebase, you can use the new, synchronous `dataRepository.SaveChanges()` method 😄